Pod/Classes/NSError+RZMutablility.h

```diff
@@ -0,0 +1,45 @@
+//
+//  NSError+RZMutablility.h
+//  RaisinToast
+//
+//  Created by alex.rouse on 8/12/14.
+//  Copyright (c) 2014 Raizlabs. All rights reserved.
+//
```

Comment:
Should probably have MIT license at the top of this file and all other source files for the repo